ui/app/store/actions.js

```diff
@@ -2428,6 +2437,24 @@ function setAutoLogoutTimeLimit (value) {
   return setPreference('autoLogoutTimeLimit', value)
 }
 
+function setThreeBoxSyncing (newThreeBoxSyncingState) {
+  return dispatch => {
+    dispatch(actions.showLoadingIndication())
+    return new Promise((resolve, reject) => {
+      background.setThreeBoxSyncing(newThreeBoxSyncingState, (err) => {
```

Comment:
`setThreeBoxSyncing` doesn't appear to exist. I encountered an error for this locally by toggling the 3Box setting.

Reply:
Similarly, the state `metamask.threeboxSyncing` doesn't exist either